thoth/messaging/unrevsolved_package.py

```diff
@@ -0,0 +1,62 @@
+#!/usr/bin/env python3
+# thoth-messaging
+# Copyright(C) 2020 Sai Sankar Gochhayat
+#
+# This program is free software: you can redistribute it and / or modify
+# it under the terms of the GNU General Public License as published by
+# the Free Software Foundation, either version 3 of the License, or
+# (at your option) any later version.
+#
+# This program is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
+# GNU General Public License for more details.
+#
+# You should have received a copy of the GNU General Public License
+# along with this program. If not, see <http://www.gnu.org/licenses/>.
+
+"""This is Thoth Messaging module for UnrevsolvedPackageMessage."""
+
+import logging
+from typing import Optional
+
+import faust
+
+from .message_base import MessageBase
+
+_LOGGER = logging.getLogger(__name__)
+
+
+class UnrevsolvedPackageMessage(MessageBase):
+    """Class used by Producer events on Kafka topic on Reverse Solver events."""
+
+    topic_name = "thoth.investigator.unrevsolved-package"
+
+    class MessageContents(faust.Record, serializer="json"):  # type: ignore
+        """Class used to represent contents of a message Kafka topic."""
+
+        package_name: str
+        package_version: Optional[str]
```

Comment:
`package_version` is mandatory to schedule revsolver: https://github.com/thoth-station/common/blob/6ee544a64d74df8136a6dd235108fdbcf8d45aa8/thoth/common/openshift.py#L854

Reply:
@pacospace thanks, I have fixed it